controllers/chat/getSignedChannelList.js

```diff
@@ -100,7 +100,7 @@ const getSignedChannelList = async (req, res) => {
   const queriedChannels = [];
   const promisedChannels = [];
 
-  const client = new StreamChat(process.env.API_KEY, process.env.API_SECRET);
+  const client = new StreamChat(process.env.API_KEY, process.env.SECRET);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `API_SECRET` to `SECRET` might cause issues if the environment variable `SECRET` is used for other purposes in your application. It's better to keep secrets specific to their usage to avoid potential conflicts or security risks.
```diff
- const client = new StreamChat(process.env.API_KEY, process.env.SECRET);
+ const client = new StreamChat(process.env.API_KEY, process.env.API_SECRET);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
The previous was wrong, so changed to SECRET